config/default.yaml

```diff
@@ -0,0 +1,8 @@
+# This is the default config for all environments. Check out custom-environment-variables.yml for all possible settings.
+
+siteUrl: http://host.docker.internal:3000
+
+algolia:
+  appId: 7AWZHGNJE2
+  apiKey: a976ea48057ceaa662656ec8f4f591af # This is a search only API key. It's safe to check in.
+  indexName: docs_site
```

Comment:
I've actually broken out a whole `stage` index - I think we'll want to have this. It will allow us to validate that new content is showing up how we want it in the search before shipping to `prod`